pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -22,12 +22,16 @@ import (
 	"io"
 	"net"
 	"net/http"
+	ossys "os"
 	"regexp"
+	"strings"
 	"time"
 
 	"code.google.com/p/gcfg"
 	"github.com/rackspace/gophercloud"
 	"github.com/rackspace/gophercloud/openstack"
+	"github.com/rackspace/gophercloud/openstack/blockstorage/v1/volumes"
+	"github.com/rackspace/gophercloud/openstack/compute/v2/extensions/volumeattach"
```

Comment:
One of these two imports is linking golang "testing" flags into all of our binaries. Not sure what to do with it, because it appears the fix will need to be made upstream. cc @lavalamp